extensions/wikia/SeoLinkHreflang/SeoLinkHreflang.class.php

```diff
@@ -50,14 +50,8 @@ private static function getMainPageLinks( $domain ) {
 		return [];
 	}
 
-	private static function getLillyLinks( $url ) {
-		$lilly = new Lilly();
-		//$url = str_replace( '.rychu.wikia-dev.com', '.wikia.com', $url );
-		return $lilly->getCluster( $url );
-	}
-
 	private static function generateHreflangLinks( OutputPage $out ) {
-		global $wgEnableLillyExt, $wgArticle;
+		global $wgArticle;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)